sdk/src/types/block/address/bech32.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use core::str::FromStr;
+use std::hash::Hash;
+
+use crate::types::block::address::Address;
+
+/// An address and its network type.
+#[derive(Clone, Eq, PartialEq, Hash)]
+pub struct Bech32Address {
+    pub(crate) inner: Address,
+    pub(crate) hrp: String,
```

Comment:
I'm bikeshedding now but I would reverse them to match the actual order of a bech32 address

Reply:
😆 okay